core/Plugin/Menu.php

```diff
@@ -30,16 +30,6 @@
  */
 class Menu
 {
-    protected $module = '';
```

Comment:
A "find usage" didn't yield any usage outside of this class but maybe @tsteur has more info